test.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+pip install pipenv
+pipenv install
+pipenv run pytest
```

Comment:
While we're here, do you want to run `black`? :smiley:

Reply:
Yeah, although maybe in a way that doesn't make uncommitted changes by accident.

Reply:
022ae61